recipes/asyncly/all/test_v1_package/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.15)
```

Comment:
The test_v1_package is no longer mandatory, you can remove if you want: https://github.com/conan-io/conan-center-index/discussions/17406

Reply:
removed